src/services/ExperimentBasic.js

```diff
@@ -88,6 +88,34 @@ export default {
             }, 4000);
             });
         },
+        setASCANCommand(device, command) {              // FOR ULTRASOUND
```

Comment:
If these two methods only for Ultrasound Experiment. You can put in Ultrasound.vue
If it will be shared with other experiments. then just keep it.